CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.11.0 - 2017-04-11
+
+- Update fury to 3.0.0-beta.1
```

Comment:
It's beta 0 not beta 1? https://github.com/apiaryio/fury.js/releases/tag/v3.0.0-beta.0